package.json

```diff
@@ -92,7 +92,7 @@
   "peerDependencies": {
     "jquery": "1.9.1 - 3",
     "popper.js": "^1.16.1",
-    "swiper": "^6.5.1",
+    "swiper": "^8.2.6",
```

Comment:
Do we need to import swiper since we import it via cdn each time we use it in the doc ?